server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMessagesCreationProcessorTest.java

```diff
@@ -179,6 +183,7 @@ protected MessageWithId createMessageInOutboxAndSend(ValueWithId.CreationMessage
         assertThat(result.getNotCreated()).isEmpty();
     }
 
+    @Ignore("JAMES-1716 : should report an error")
```

Comment:
yes, I was just asking because I thought that was a specific already handled case (mailboxrolenotfoundexception)